recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
Just two suggestions: 

 - maybe point out that the order of channels matters – I had `conda-forge` with lower priority than `defaults` which resulted in an error (changing priority resulted in `conda:     4.3.13-py27_0 --> 4.2.13-py27_0 conda-forge` and `conda-env: 2.6.0-0       --> 2.6.0-0       conda-forge`)
 - Are the CI tokens (see below) necessary? Conda-smithy asked for them, but by the time I had fixed the channel order I had already added the tokens so I don't know if the command would have worked without them.

```
$ conda smithy rerender
No circle token.  Create a token at https://circleci.com/account/api and
put it in ~/.conda-smithy/circle.token
No appveyor token. Create a token at https://ci.appveyor.com/api-token and
Put one in ~/.conda-smithy/appveyor.token
No anaconda token. Create a token via
  anaconda auth --create --name conda-smithy --scopes "repos conda api"
and put it in ~/.conda-smithy/anaconda.token
Fetching package metadata .............
```